groups/managers.py

```diff
@@ -1,10 +1,53 @@
+import datetime
+
+from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.db import models
 from polymorphic import PolymorphicManager, PolymorphicQuerySet
 
 
-class GroupQuerySet(models.QuerySet):
+DEFAULT_WITHIN_DAYS = apps.get_app_config('groups').default_within_days
+
+
+class WithinDaysUserQuerySetMixin:
+    """
+    A mixin that adds methods for returning items that have recently been posted (to).
+
+    Can be mixed into a Manager or a QuerySet.
+
+    By default, is intended for use with User objects.  To change this, override the
+    condition used in the `since()` method, which is a member called `since_filter`.
+    """
+    since_filter = 'comments__date_created__gte'
```

Comment:
I wanted a default so people could just add `WithinDaysUserQuerySetMixin` to their managers for `User` without having to faff around.